python/tskit/trees.py

```diff
@@ -1425,6 +1429,50 @@ def get_parent_dict(self):
     def __str__(self):
         return str(self.get_parent_dict())
 
+    def map_mutations(self, genotypes):
+        """
+        Given observations for a discrete set of characters at the samples in this
+        tree, reconstruct the state transitions required to reproduce these
+        observations. The observations are described by the genotypes array, with
+        distinct characters mapped to integers in the range 0 to 63,
+        such that ``genotypes[0]`` contains the observation for sample 0 and so on.
+        Missing data can be specified for a sample using the value
+        ``tskit.MISSING_DATA`` (-1). At least one non-missing observation must be
+        provided.
+
+        The current implementation uses the Fitch parsimony algorithm to determine
+        the minimum number of state transitions required to explain the data.
+
+        The state reconstruction is returned as two-tuple, ``(ancestral_state,
+        transitions)``, where ancestral_state is the integer state that has been
+        determined to be most parsimonious. The returned ``transitions`` is a
+        list of ``tskit.StateTransition`` objects, which correspond to
+        mutations in the :ref:`sec_data_model`. Each element in this list
+        in an object with attributes ``node``, ``parent`` and ``state``,
+        which alternatively can be treated as a 3-tuple (it is implemented using
+        :func:`collections.namedtuple`). For each transition, ``node`` is the
+        tree node over which state transition occurs and ``state`` is the new
+        after this transition. When there are multiple transitions at returned,
+        ``parent`` contains the index of the previous state transition on the path
+        to root (see the :ref:`sec_mutation_table_definition` for more information
+        on the concept of mutation parents).
```

Comment:
Misisng word: "and ``state`` is the new **state** after this transition".

Also, the mutation being "over" a node is a bit ambiguous, how about "``node`` is the tree node at the bottom of the branch on which the transition occurred"?